src/main/java/svc/models/DemoCitation.java

```diff
@@ -0,0 +1,11 @@
+package svc.models;
+
+import java.time.LocalDate;
+
+public class DemoCitation {
+	public String citationNumber;
+	public LocalDate dob;
+	public String driversLicenseNumber;
+	public Integer numberOfViolations;
+	public Integer daysTillCourt;
```

Comment:
It adds more unnecessary fragmentation but as long as we get this code out ASAP then it's fine.